partner_autocomplete_disable/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class ResPartnerAutocompleteDisable(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def autocomplete(self, query):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-partner-addons/pullRequest?prid=5363009)